core/arch/x86/decode_table.c

```diff
@@ -2140,18 +2140,18 @@ const instr_info_t second_byte[] = {
   {INVALID, 0x0f3f10, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   /* 40 */
   {OP_cmovo,   0x0f4010, "cmovo",  Gv, xx, Ev, xx, xx, mrm|predcc, fRO, END_LIST},
-  {E_VEX_EXT, 0x0f4110, "(vex ext 83)", xx, xx, xx, xx, xx, mrm, x, 83},
-  {E_VEX_EXT, 0x0f4210, "(vex ext 84)", xx, xx, xx, xx, xx, mrm, x, 84},
+  {E_VEX_EXT, 0x0f4110, "(e_vex ext 83)", xx, xx, xx, xx, xx, mrm, x, 83},
```

Comment:
Optional: Might be worth doing this e_vex separately: certainly cleaner that way, as this belongs with the prior commit.  It is a bit confusing to have one commit doing the rename to E_VEX_EXT with this part of the rename buried in this larger commit.  If not separating, have a separate paragraph in the commit message referencing the prior commit hash.

Reply:
I'll add the paragraph. This time I'd like to get this into upstream so that outside contributors can start adding to it, if possible.